_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -0,0 +1,161 @@
+
+**Important**: The impelementation only uses the
+`config.istio.io/pilotCfgStore` and `config.istio.io/accessConfigFile`
+annotations, although every other annotation and spec is validated for
+correct syntax.  They may be set to dummy values, as long as they can
+be validated correctly.
+
+If the prequisites are met, the credentials for each Kubernetes cluster
+will also be present in `$HOME/multicluster`.
+
+Assemble and create the configmap from these files in the Kubernetes cluster
+where the Istio control plane is operational:
+
+```bash
+kubectl create configmap -n istio-system multicluster -f $HOME/multicluster
+```
+
+## Deploy on the remote cluster
+
+Multicluster makes **one** Istio service mesh from all Kubernetes clusters
+that participate in multicluster.  Multicluster uses **one** Kubernetes
+cluster to run the Istio control plane.  **Any reasonable** number of remote
+Kubernetes clusters may be attached to the **one** Istio control plane.
+
+### Use kubectl with Helm to connect the remote cluster
+
+1. Use the helm template command on a remote to specify the Istio control plane:
+
+   ```bash
+   helm template install/kubernetes/helm/istio-remote --name istio-remote --set pilotEndpoint=`pod_ip_of_pilot_master` --set mixerEndpoint=`pod_ip_of_mixer_master` > $HOME/istio-remote.yaml
+    ```
+
+1. Instantiate the remote cluster's connection to the Istio control plane:
+
+   ```bash
+   kubectl create -f $HOME/istio-remote.yaml
+   ```
+
+### Use Helm and Tiller to connect the remote cluster
+
+1. If a service account has not already been installed for Helm, please
+install one:
+
+   ```bash
+   kubectl create -f install/kubernetes/helm/helm-service-account.yaml
+   ```
+
+1. Initialize Helm:
+
+   ```bash
+   helm init --service-account tiller
+   ```
+
+1. Install the Helm chart:
+
+   ```bash
+   helm install install/kubernetes/helm/istio-remote --name istio-remote --set pilotEndpoint=pod_ip_of_pilot_master --set mixerEndpoint=pod_ip_of_mixer_master`
+   ```
+
+### Mandatory Helm configuration parameters
+
+The isito-remote Helm chart requires the configuration of two specific variables defined in the following table:
+
+**Note** The `pilotEndpoint` and `mixerEndpoint` can also use DNS resolution, assuming DNS is setup to resolve
+these IPs.
+
+| Helm Variable | Accepted Values | Default | Purpose of Value |
+| --- | --- | --- | --- |
+| `global.pilotEndpoint` | A valid IPv4 address | none | Specifies the Istio control plane's pilot Pod IP address |
+| `global.mixerEndpoint` | A valid IPv4 address | none | Specifies the Istio control plane's mixer Pod IP address |
+
+## Uninstalling
+
+** Note the uninstallation method must match the installation method (Helm or kubectl based) **
+
+### Using kubectl to uninstall the istio-remote
+
+* Uninstall an Istio remote:
+
+  ```bash
+  kubectl delete -f $HOME/istio-remote.yaml
+  ```
+
+### Using tiller to uninstall the istio-remote
+
+  ```bash
+  helm delete --purge istio-remote
+  ```
```

Comment:
I think we need some way to point out limitations and issues.   Especially the mixer limitation on learning of the remote service naming.   Also istio-ca isn't multicluster aware.   I would prefer a section here,  but at the least we should create issues in istio that we can reference here.

Reply:
An issue sounds good.  Please file one in the istio/istio repo and this repo with what you have in mind for improvements.  This simply documents the current state of the implementation and I don't have a strong desire to create sea salt with this PR.

Reply:
What is your plan with Caveats?   Mixer is not multicluster capable and neither is istio-ca.  Should we just create issue to not those?